thinc/layers/pytorchwrapper.py

```diff
@@ -11,7 +11,7 @@
 
 @registry.layers("PyTorchRNNWrapper.v1")
 def PyTorchRNNWrapper(
-    pytorch_model,
+    pytorch_model: "torch.nn.Module",
```

Comment:
Different types in the new layer is fine, but I might err on the side of not modifying the types in the existing layers?

Reply:
Fixed.